pkg/cli/cmds/root.go

```diff
@@ -62,11 +64,10 @@ func init() {
 // kernel.panic_on_oops=1
 // kernel.keys.root_maxbytes=25000000
```

Comment:
drop maxbytes from comment

Reply:
dropped all args from comments, no reason to duplicate them